webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7355,44 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.getBotDetails = function () {
+                window.Applozic.ALApiService.ajax({
+                    url: MCK_BOT_API + "/bot/" + MCK_APP_ID,
```

Comment:
@hardikparashar why are we fetching all bots of the application, not directly the assigned one?

Reply:
I checked, found no API that fetched a single bot.

Reply:
What is stopping you from developing one :)

Reply:
the api has been created